SU-HKKU/cardinal_service_profile
# CSD-304 Use views field view to make distinct resource results

# READY FOR REVIEW

# Summary
- Use views_field_view to produce distinct results
- Adjust the view and include styling for resource list.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. make sure the stack repo has been updated, pull the latest and `composer install -n`
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-304_distinct`
1. `blt drupal:sync --site=cardinalservice`
1. view the `/resources` page and verify there are no duplicates.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
